src/scroll.js

```diff
@@ -0,0 +1,66 @@
+import smoothscroll from 'smoothscroll-polyfill';
+
+smoothscroll.polyfill();
+
+const AlpineScrollMagicMethod = {
+    start() {
+
+        Alpine.addMagicProperty('scroll', function ($el) {
+            return function (target, options = {}) {
+                const originalTarget = target
+
+                // Check if we specified an offset
+                const offset = options['offset'] ? parseInt(options['offset'], 10) : 0
+                delete options.offset
+
+                // Support integers specified as strings
+                // We do a strict check first because we don't whant to support things like "100foo"
+                if (typeof target === 'string' && /^[0-9]+?/g.test(target)) {
+                    target = parseInt(target, 10)
+                }
+
+                // Support for CSS query selector
+                let element = null
```

Comment:
@SimoTod While merging scroll into range ES lint it looks like `element` isn't used. I'll just remove it but can you check if it should have been used elsewhere? 

I feel like you shouldn't have been able to push it without being warned, right? Do we have something not set up correctly? (Also, some semicolons got in as well)

Reply:
Yeah, element can be removed. The linter did run after I merged master (there were some conflicts, I had to fix them and eslint complained) 🤷‍♂️
I'll double check I don't have some weird local setting next time.